include/ProcessTable.h

```diff
@@ -1,10 +1,12 @@
-#ifndef PROCESS_TABLE_H
-#define PROCESS_TABLE_H
+#pragma once
+
+#include <stdint.h>
 
 #include <iomanip>
 #include <string>
 #include <vector>
 
+namespace proctable {
```

Comment:
вкусовщина: рекомендую отделять неймспейс пустой строкой, чтобы не сливались внутренние объекты, а то они индентацией не выделяются и иногда тяжело смотреть)

```suggestion
namespace proctable {

```